### ISSUE TITLE
NUL bytes in commented lines

### ISSUE THREAD

**Issue by 7aa6e20b-8983-474f-b2ae-de7eff1caa04** [python/cpython#64314]
BPO | [20115](https://bugs.python.org/issue20115)
--- | :---
Nosy | @gvanrossum, @arigo, @birkenfeld, @terryjreedy, @benjaminp, @jwilk, @alex, @serhiy-storchaka, @iritkatriel

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2014-01-03.17:59:13.390>
labels = ['interpreter-core', 'type-bug', '3.9', '3.10', '3.11']
title = 'NUL bytes in commented lines'
updated_at = <Date 2021-09-07.05:32:27.769>
user = 'https://github.com/arigo'
```

bugs.python.org fields:
```python
activity = <Date 2021-09-07.05:32:27.769>
actor = 'gvanrossum'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Interpreter Core']
creation = <Date 2014-01-03.17:59:13.390>
creator = 'arigo'
dependencies = []
files = []
hgrepos = []
issue_num = 20115
keywords = []
message_count = 19.0
messages = ['207232', '207282', '207290', '207358', '207872', '207873', '207879', '207939', '208086', '208087', '218239', '395928', '395932', '395938', '395989', '401193', '401196', '401204', '401208']
nosy_count = 11.0
nosy_names = ['gvanrossum', 'arigo', 'georg.brandl', 'terry.reedy', 'benjamin.peterson', 'jwilk', 'Arfrever', 'alex', 'ita1024', 'serhiy.storchaka', 'iritkatriel']
pr_nums = []
priority = 'low'
resolution = None
stage = 'needs patch'
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue20115'
versions = ['Python 3.9', 'Python 3.10', 'Python 3.11']
```

</p></details>

**Comment by asottile** [comment]
I believe this can be closed as fixed #96670

**Comment by iritkatriel** [comment]
Agreed, this is fixed.

```
% ./python.exe x.py 
  File "/Users/iritkatriel/src/cpython-1/x.py", line 1
    #
SyntaxError: source code cannot contain null bytes
```
